datasets/xtreme/xtreme.py

```diff
@@ -450,12 +450,46 @@ def _info(self):
             features["label"] = datasets.Value("string")
         if self.config.name == "XNLI":
             features["gold_label"] = datasets.Value("string")
+        
+        if self.config.name.startswith("udpos"):
+            features = datasets.Features(
+                {
+                    "token": datasets.Value("string"),
```

Comment:
`tokens` here ?

Reply:
No, here they are single token, not a sequence.

Reply:
Yes right !